Bylaws.md

```diff
@@ -194,37 +212,37 @@ The Forum shall maintain a Member Mail List and Member Web Site that are not acc
 
 Members have discretion about which mailing list they use, but are strongly encouraged to use the Public Mail List for matters other than those listed above.
 
-Members are strongly discouraged from posting the text of Member Mail List messages to the Forum Public Mail List without the permission of the author or commenter.
+Members are strongly discouraged from posting the text of Member Mail List messages to a Public Mail List without the permission of the author or commenter.
```

Comment:
I suggest further modifying this line to connect it to the requirements of the Code of Conduct. I would propose the following, but am not particularly married to this specific text:

`Members are strongly discouraged from reposting the text of Member Mail List messages without the permission of the author or commenter. Doing so may violate the Code of Conduct.`

Reply:
Updated https://github.com/cabforum/forum/pull/32/commits/3c3a1aa55886ff217ac9c808f96a5e9a9582fc11